patterns/cell-based-eks/0.vpc/main.tf

```
@@ -0,0 +1,47 @@
+provider "aws" {
+  region = local.region
+}
+
+data "aws_availability_zones" "available" {}
+
+locals {
+  cluster_name = format("%s-%s", basename(path.cwd), "shared")
```

Lets follow the current norm from what is used in other patterns:

```suggestion
  name   = basename(path.cwd)
```

---

Done